Project-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js

```diff
@@ -1,2 +1,14 @@
-import React from 'react';
-import './Button.css';
+import React from "react";
+
+const ActionButton = props => {
+  return (
+    <div
+      className={`button action-button ${props.className}`}
+      style={props.style}
```

Comment:
rather than trying to add inline styles here like we sometimes would with DOM elements, make this add any passed-down `style` prop as an interpolated expression inside the ActionButton className like `className` is in the line above, or just add the color to the passed-in className where you pass it in in ButtonsGroup.js. then in your less/css you can add a selector for anything with the class `turqoise` for example, and say the background color of the selected elements should be turqoise